code-samples/deployment/docker/build-a-dockerfile/commands/prune.js

```diff
@@ -0,0 +1,21 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
+
+module.exports = {
+	data: new SlashCommandBuilder()
+		.setName('prune')
+		.setDescription('Prune up to 99 messages.')
+		.addIntegerOption(option => option.setName('amount').setDescription('Number of messages to prune')),
```

Comment:
```suggestion
		.addIntegerOption(option => option.setName('amount').setDescription('Number of messages to prune').setMinValue(1).setMaxValue(100).setRequired(true)),
```
The if statement in the code can then be removed

Reply:
This was directly copied from the `creating-your-bot` section